imitation/util/rollout.py

```diff
@@ -437,3 +439,75 @@ def generate_transitions_multiple(policies, env, n_timesteps, *, truncate=True,
     assert len(act) == n_timesteps
 
   return tuple(np.array(x) for x in (obs_old, act, obs_new))
+
+
+def save_transitions(rollout_dir: str,
```

Comment:
Why are we saving transitions rather than trajectories? You can always flatten trajectories to transitions, but you cannot recover trajectories from transitions. Sinec some algorithms operate on trajectories not transitions it'd be nice to not lose this information when saving. @qxcv?

A format that allows for the most compression is to just add a bitmask "mask" or "done" flag, indicating whether it's the end of an episode. This is used in Stable Baselines GAIL. It's a bit clunky to work with though, so saving our list-of-trajectories format would be simpler. https://github.com/hill-a/stable-baselines/issues/281 has a discussion of pros and cons.

Reply:
Agreed that saving transitions instead is a good idea. This should also help with #48 because we'd be close to being able to infer the number of episodes to average over from the dump.